src/NTDll.Desktop/NTDll.cs

```diff
@@ -23,5 +23,25 @@ public static partial class NTDll
         /// </remarks>
         [DllImport(nameof(NTDll))]
         public static extern Win32ErrorCode RtlNtStatusToDosError(NTSTATUS Status);
+
+        /// <summary>
+        /// The NtOpenSection routine opens a handle for an existing section object.
+        /// </summary>
+        /// <param name="sectionHandle">Pointer to a HANDLE variable that receives a handle to the section object.</param><param name="desiredAccess">Specifies an ACCESS_MASK value that determines the requested access to the object.</param><param name="objectAttributes">Pointer to an OBJECT_ATTRIBUTES structure that specifies the object name and other attributes. Use InitializeObjectAttributes to initialize this structure.</param>
+        /// <returns>
+        /// Returns <see cref="NTSTATUS.Code.STATUS_SUCCESS"/> on success, or the appropriate <see cref="NTSTATUS"/> error code on failure.
+        /// </returns>
+        [DllImport(nameof(NTDll), CharSet = CharSet.Unicode)]
+        public static extern NTSTATUS NtOpenSection([Out] out IntPtr sectionHandle, [In] uint desiredAccess, [In, Out] ref OBJECT_ATTRIBUTES objectAttributes);
```

Comment:
* I don't think that `[In]` or `[Out]` are necessary here.
* `objectAttributes` can be a pointer using the friendly attributes to generate the ref overload
* `objectAttriubtes` seem to be In only in https://msdn.microsoft.com/en-us/library/windows/hardware/ff567029(v=vs.85).aspx